pkgs/development/python-modules/eth-account/default.nix

```
@@ -16,15 +16,15 @@
 
 buildPythonPackage rec {
   pname = "eth-account";
-  version = "0.9.0";
+  version = "0.13.3";
   format = "setuptools";
```

```suggestion
  pyproject = true;
```
and specify `build-system`.